packages/core/app-extensions/src/rest/helpers.js

```diff
@@ -12,17 +12,27 @@ import {requestSaga} from './rest'
  * @param entityName {String} Name of the entity
  * @param key {String} key of the entity
  * @param query {Object} see 'buildRequestQuery' function
+ * @param requestOptions {Object} An object which can contain the following options:
+ * - method {String} HTTP Method of request. Default is POST.
  * @param transformer {function} Function to directly manipulate the result. By default returns data attribute
  */
-export function* fetchEntity(entityName, key, query, transformer = json => json) {
+export function* fetchEntity(entityName, key, query, {method = 'POST'} = {}, transformer = json => json) {
```

Comment:
Destructuring and Default Assignment both in the parameter list is pretty neat, but I wonder if we should just do this in the method itself to keep it easily understandable in the future?

Reply:
absolutely right 👍  moved it into the method body.